pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/NamespacesTest.java

```diff
@@ -1748,4 +1750,68 @@ private void assertInvalidRetentionPolicyAsPartOfAllPolicies(Policies policies,
             assertTrue(e.getMessage().startsWith("Invalid retention policy"));
         }
     }
+
+    @Test
+    public void testExistNamespaceResourceGroup() {
+        String namespace = BrokerTestUtil.newUniqueName(this.testTenant + "/namespace");
+        ResourceGroup testResourceGroup = new ResourceGroup();
+        testResourceGroup.setDispatchRateInBytes(10000L);
+        testResourceGroup.setDispatchRateInMsgs(100);
+        testResourceGroup.setPublishRateInMsgs(100);
+        testResourceGroup.setPublishRateInBytes(10000L);
+        final String resourceGroupName = "test-resource-group";
+        try {
+            admin.resourcegroups().createResourceGroup(resourceGroupName, testResourceGroup);
+            admin.namespaces().createNamespace(namespace);
+            admin.namespaces().setNamespaceResourceGroup(namespace, resourceGroupName);
+            String namespaceResourceGroup = admin.namespaces().getNamespaceResourceGroup(namespace);
+            assertEquals(namespaceResourceGroup, resourceGroupName);
+            admin.namespaces().removeNamespaceResourceGroup(namespace);
+            assertTrue(StringUtils.isBlank(admin.namespaces().getNamespaceResourceGroup(namespace)));
```

Comment:
We should call this in the `Awaitility.await().untilAsserted()`.

Reply:
Could you give me more information about why use it? 
it's a sync method, I think we don't need to use ``Awaitility ``.